### ISSUE TITLE
Failed to load the checkpoint file provided in the google drive.

### ISSUE THREAD

**Issue by Irlyue** [KupynOrest/DeblurGAN#145]
The repo has already provided a `latest_net_G.pth` in the folder `./checkpoints/experiment_name`. However, as I go through the readme file and download the checkpoint file from the [google drive](https://drive.google.com/file/d/1liKzdjMRHZ-i5MWhC72EL7UZLNPj5_8Y/view?usp=sharing), I cannot load it into the `resnet_9blocks` generator. It got some unexpected keys in the state_dict:
>         Unexpected key(s) in state_dict: "model.10.conv_block.6.weight", "model.10.conv_block.6.bias", "model.10.conv_block.7.running_mean", "model.10.conv_block.7.running_var", "model.11.conv_block.6.weight", "model.11.conv_block.6.bias", "model.11.conv_block.7.running_mean", "model.11.conv_block.7.running_var", "model.12.conv_block.6.weight", "model.12.conv_block.6.bias", "model.12.conv_block.7.running_mean", "model.12.conv_block.7.running_var", "model.13.conv_block.6.weight", "model.13.conv_block.6.bias", "model.13.conv_block.7.running_mean", "model.13.conv_block.7.running_var", "model.14.conv_block.6.weight", "model.14.conv_block.6.bias", "model.14.conv_block.7.running_mean", "model.14.conv_block.7.running_var", "model.15.conv_block.6.weight", "model.15.conv_block.6.bias", "model.15.conv_block.7.running_mean", "model.15.conv_block.7.running_var", "model.16.conv_block.6.weight", "model.16.conv_block.6.bias", "model.16.conv_block.7.running_mean", "model.16.conv_block.7.running_var", "model.17.conv_block.6.weight", "model.17.conv_block.6.bias", "model.17.conv_block.7.running_mean", "model.17.conv_block.7.running_var", "model.18.conv_block.6.weight", "model.18.conv_block.6.bias", "model.18.conv_block.7.running_mean", "model.18.conv_block.7.running_var".

Could it be the version problem of Pytorch? (Mine is 0.4.1)

**Comment by ak9250** [comment]
getting same problem

**Comment by Jambalaya11** [comment]
+1

**Comment by QWERDF007** [comment]
+1

**Comment by furuit** [comment]
+1

**Comment by qingsenyangit** [comment]
+1

**Comment by SeaOfOcean** [comment]
+1

**Comment by kregmi** [comment]
+1

**Comment by MrBurnst** [comment]
+1

**Comment by Madrid-Ista** [comment]
+1

**Comment by Jarvisss** [comment]
any resolutions?

**Comment by doduythao** [comment]
so do I

**Comment by Muzammal-Naseer** [comment]
+1

**Comment by umar1992** [comment]
I am facing the same issue..my Pytorch version is 1.0.1. Anyone could find any solution?

**Comment by leandepk** [comment]
have solution？？？

**Comment by 96lives** [comment]
It seems like one commit changed the architecture of ResNet!
Erase `(repo root)/models/networks.py` line 219 ~ 230 and add
 ` if use_dropout:
          blocks += [nn.Dropout(0.5)]
   blocks += padAndConv[padding_type] + [norm_layer(dim)]
`
to make the code work.

However, I don't seem to get the unblurry result with the pretrained model....

**Comment by idealboy** [comment]
＋１
